### DISCUSSION TITLE
Code reference based on mkdocs, mkdocs-material, mkdocstrings

### DISCUSSION THREAD

**Discussion by pawamoy** [qtile/qtile#3940] [Show and tell]
Following the conversation in https://github.com/qtile/qtile/pull/3540, I wanted to see how the API docs would look like within a Material for MkDocs site :slightly_smiling_face: 
I deployed it on my fork's pages, here's the documented `bar` module: https://pawamoy.github.io/qtile/reference/libqtile/bar/
I built it from master: docstrings are not yet normalized to a particular style it seems (mix of sphinx and numpy).
Well, that's it, that was fun :smile:

**Comment by m-col** [comment]
Pretty neat :)
